resource/airflow_dag_pipeline.yaml

```diff
@@ -121,6 +121,47 @@ jobs:
 
   # test creating the IAM role
   - aggregate:
+    - task: check-diff-in-iam-role
+      privileged: true
```

Comment:
are these `privileged: true` required? I thought they were only needed if your task was building an image

Reply:
Definitely not needed for this so have removed